From 8d761383e0bdb9ce0d4c3f4efe1d14ffc0e06ba4 Mon Sep 17 00:00:00 2001
From: Kai Kang <kai.kang@windriver.com>
Date: Mon, 2 Aug 2021 16:25:42 +0800
Subject: [PATCH] anaconda.py: use 'Fedora' as default product

It used to use 'Fedora' as default product in Anaconda 32. But it
changed to get from /etc/os-release file. Set it with 'Fedora' directly
for oe.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 37.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 anaconda.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/anaconda.py b/anaconda.py
index 2b3b42d19a..441309d816 100755
--- a/anaconda.py
+++ b/anaconda.py
@@ -208,8 +208,7 @@ if __name__ == "__main__":
         )
     else:
         conf.set_from_detected_profile(
-            util.get_os_release_value("ID"),
-            util.get_os_release_value("VARIANT_ID"),
+            'fedora'
         )
 
     conf.set_from_files()
-- 
2.25.1

